head/src/main/kotlin/io/qalipsis/core/head/web/DataSeriesController.kt

```diff
@@ -370,4 +370,24 @@ internal class DataSeriesController(
         )
     }
 
+    @Patch("/refresh-all")
+    @Operation(
+        summary = "Refresh the prepared queries of the data series",
+        description = "Update the prepared queries of all the data series",
+        responses = [
+            ApiResponse(responseCode = "200", description = "Data series refreshed"),
+            ApiResponse(responseCode = "401", description = "Operation not allowed"),
+        ],
+        security = [
+            SecurityRequirement(name = "JWT")
+        ]
+    )
+    @Secured(Permissions.WRITE_DATA_SERIES_PREPARED_QUERIES)
+    @Timed("data-series-refresh")
+    suspend fun refresh(
```

Comment:
Please return a 202 (accepted) instead of a 200 when the operation is asynchronous.